### DISCUSSION TITLE
Are live captions supposed to work?

### DISCUSSION THREAD

**Discussion by Aedon** [Alex313031/Thorium-Win-AVX2#85] [General]
Are Live captions supported in M112?

**Comment by gz83** [comment]
@Aedon

You can find this feature in the settings page, see the screenshot below

![image](https://github.com/Alex313031/Thorium-AVX2/assets/17492865/790d9129-22eb-4a65-99b7-ad08d9764b49)

**Comment by Aedon** [comment]
That doesn't work, comparing it to normal chrome where it displays captions:
![thorium axv2](https://github.com/Alex313031/Thorium-AVX2/assets/72438130/42803e5f-1756-4c6c-bc22-f5063be06f9c)

![chrome](https://github.com/Alex313031/Thorium-AVX2/assets/72438130/494d8268-0920-46b5-8383-efe343aa9c14)

**Comment by gz83** [comment]
@Aedon 

Does version 111 have this problem?

**Reply by Aedon** [reply]
I tried the portable 111 avx2, it exhibited the same behaviour.

**Comment by Alex313031** [comment]
@gz83 @Aedon This is a known bug in Thorium, but it is not thorium specific. Rather, it affects any chromium build. While live caption is enabled, and api keys are enabled, the actual code that performs [SODA](https://chromium.googlesource.com/chromium/src/+/master/components/soda/) is internal only. Chromium forks that have live caption enabled (like brave) have had to write their own implementations/replacements. Something I dont have the knowledge or time to do. Sorry.
